py/picca/bal_tools.py

```diff
@@ -0,0 +1,67 @@
+##Functions used for masking BAL features ##LE 2020
+
+
+import fitsio
+import scipy as sp
+
+def read_bal(bal_catalog): ##Based on read_dla from picca/py/picca/io.py
+    lst = ['THING_ID','VMIN_CIV_450','VMAX_CIV_450','VMIN_CIV_2000','VMAX_CIV_2000']
+    h = fitsio.FITS(bal_catalog)
+    bal_dict = { k :h[1][k][:] for k in lst }
+    h.close()
+
+    return bal_dict
+
+
+#based on add_dla from picca/py/picca/data.py
+def add_bal_rf(bal_catalog,thingid,bal_index): #BAL catalog, THING_ID, and BAL index
+    ### Wavelengths in Angstroms
+    lines = {
+        "lCIV" : 1549, #Used for testing
+        "lNV" : 1240.81,
+        "lLya" : 1216.1,
+        "lLyb" : 1020,
+        "lOIV" : 1031,
+        "lOVI" : 1037,
+        "lOI" : 1039
+    }
+
+    if bal_index == 'bi':
+        lst = ['VMIN_CIV_2000','VMAX_CIV_2000']
+    else: ##AI, the default
+        lst = ['VMIN_CIV_450','VMAX_CIV_450']
+
+    bal_mask_rf = []
+
+    ls = sp.constants.c*10**-3 ##Speed of light in km/s
+
+    vMin = [] ##list of minimum velocities
```

Comment:
Rename `vMin` to follow standard variable naming. It would also be helpful if the name was self-descripting. I would suggest `min_velocities`.
Similarly for `vMax`